### ISSUE TITLE
Calls not using bluetooth headphones on multiple devices

### ISSUE THREAD

**Issue by WardPearce** [oxen-io/session-android#1084]
**Code of conduct**

- [X] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-android/blob/master/CODE_OF_CONDUCT.md).

**Describe the bug**

When I start a call on any of my Android devices or any of my friends devices with Bluetooth headphones, call audio often still goes through speaker.

**To reproduce**

Connect headphones, start call.

**Smartphone (please complete the following information):**

 - Device: Google Pixel 6, Samsung Flip & other devices of friends.
 - OS: Android 
 - 1.16.3

**Comment by lordme-1** [comment]
I think I have the same issue with my phone. 
I downloaded session today off of the Google store. I have a Samson galaxy s20 FE 5G. I tried it with aptx Codec and SBC Codec. 

![Screenshot_20230301_180609_Settings](https://user-images.githubusercontent.com/52361124/222287144-b8fe3098-be34-420f-b20a-4e5338b5da85.jpg)

![Screenshot_20230301_180831_Settings](https://user-images.githubusercontent.com/52361124/222287368-19f1165f-e4fb-4092-b12d-c8d6f8e1f3bf.jpg)

![Screenshot_20230301_181104_Settings](https://user-images.githubusercontent.com/52361124/222287755-a3ab9c22-d2be-4ea4-b108-c9891723f32c.jpg)

**Comment by farline99** [comment]
Yes, the same thing. The microphone and phone sound are used, not headphones.

**Comment by x80486** [comment]
I reported this a while back as well. No progress whatsoever. I'm unable to connect via Bluetooth as well.

**Comment by KeeJef** [comment]
A fix is coming for this in the upcoming point release

**Comment by WardPearce** [comment]
> A fix is coming for this in the upcoming point release

Awesome, great to hear. Thank you!

**Comment by Aaron-von-Awesome** [comment]
> A fix is coming for this in the upcoming point release

What release with that happen to be?  Last two did not resolve :-(

**Comment by therealfakegamerboy** [comment]
I am experiencing this issue in 2024, also on Pixel 6.

**Comment by KeeJef** [comment]
I think we did include a fix which helped resolve this issue or a related issue for some subset of users. However looks like it still remains an issue, tracking this internally via https://optf.atlassian.net/browse/SES-1451
